.github/workflows/ci.yaml

```diff
@@ -0,0 +1,76 @@
+name: ci
+
+on: [push]
+
+env:
+  REGISTRY: justmark0
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.10"]
```

Comment:
No need to use the matrix unless you have multiple configurations that need to be run.